packages/core/lib/console-child.js

```diff
@@ -23,41 +24,71 @@ global.crypto = {
 const { network, config, url } = yargs(input[0]).argv;
 const detectedConfig = Config.detect({ network, config });
 
-function getConfiguredNetworkUrl(customConfig) {
+function getConfiguredNetworkUrl(customConfig, isDashboardNetwork) {
+  const defaultPort = isDashboardNetwork
+    ? managedDashboardDefaultPort
+    : managedGanacheDefaultPort;
   const configuredNetworkOptions = {
-    host: customConfig.host || managedGanacheDefaultHost,
-    port: customConfig.port || managedGanacheDefaultPort
+    host: customConfig.host || defaultHost,
+    port: customConfig.port || defaultPort
   };
-  return `http://${configuredNetworkOptions.host}:${configuredNetworkOptions.port}/`;
+  const urlSuffix = isDashboardNetwork ? "/rpc" : "";
+  return `http://${configuredNetworkOptions.host}:${configuredNetworkOptions.port}${urlSuffix}`;
 }
 
 let configuredNetwork;
 
-if (url) {
+const configNetworkWithProvider =
```

Comment:
Is `configDefinesProvider` clearer than `configNetworkWithProvider`?

Reply:
Your suggestion makes sense. `configDefinesProvider` sounds like a better alternative.